app/controllers/sessions_controller.rb

```diff
@@ -1,10 +1,15 @@
 class SessionsController < ApplicationController
   def new; end
 
+  def remember
+    params[:session][:remember_me] == "1" ? remember(user) : forget(user)
+  end
+
   def create
     user = User.find_by email: params[:session][:email].downcase
     if user&.authenticate params[:session][:password]
       log_in user
+      SessionsController.remember
```

Comment:
sao phải dùng như thế này

Reply:
neu khong dua vao method thi khi run rubocop se bao loi : Assignment Branch Condition size for create is too high.
em lam nhu vay de giam di 1 size cua branch condition